.github/actions/build_apko/action.yml

```
@@ -0,0 +1,61 @@
+name: Build apko image
+description: Build an apko image based on a supplied .yaml file
+
+inputs:
+  apkoConfig:
+    description: "Path to the apko .yaml config file. If left empty, all images will be built."
+    required: false
+  registry:
+    description: "Container registry to use"
+    default: "ghcr.io"
+    required: true
+  githubToken:
+    description: "GitHub authorization token"
+    required: true
+
+# Linux runner only (podman required)
+runs:
+  using: "composite"
+  steps:
+    - name: Install podman
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y podman
+
+    - name: Log in to the Container registry
+      id: podman-login
+      uses: redhat-actions/podman-login@v1
```

Dependent actions should be referenced by hash with version as comment.

```suggestion
      uses: redhat-actions/podman-login@7283c5d4ffb37e87a455332f0d68b144536d842f # v1.5
```